pkg/cmd/server/rhoai-normalizer/controller.go

```diff
@@ -216,9 +216,13 @@ func (r *RHOAINormalizerReconcile) Reconcile(ctx context.Context, request reconc
 
 	if err != nil {
 		log.V(4).Info(fmt.Sprintf("initiating delete processing for %s", name.String()))
-		//TODO initiate delete processing, interacting with catalog rest API directly, our extensions are not involved;
-		//TODO or we just wait for the polling entity provide to see things are gone and delete
-		//TODO minimally we need to delete from our location service / our storage
+		// now, delete of the inference service does not mean the model has been deleted from model registry,
+		// so we don't remove the model from our catalog, but may simply remove the URL associated with the inference
```

Comment:
What will trigger a deletion of the model from the catalog?

Reply:
Will that happen when we poll KFMR? If the model is no longer present, then we handle the deletion from the storage/location services?

Reply:
though I think you may have sorted it out per you comment https://github.com/redhat-ai-dev/model-catalog-bridge/pull/6#issuecomment-2711739094 ... that said, a better comment always helps